src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -788,6 +788,18 @@ public function testNestedFoldedStringBlockWithComments()
         ));
     }
 
+    public function testFoldedBinaryBlock()
```

Comment:
`testBinaryTag` sounds more reasonable to me and I would pass the different YAML strings through a data provider.